proxies/nodejs/handlers/location.ts

```diff
@@ -0,0 +1,95 @@
+import Koa from "koa";
+import { DVCClient, DVCUser } from "@devcycle/nodejs-server-sdk";
+
+type LocationRequestBody = {
+  command: string;
+  params: string; //[{value: string | number | boolean} | {location: string} | {callbackUrl: string}] HTTP request comes in as string
+  isAsync: boolean;
+};
+
+export const handleLocation = async (
+  ctx: Koa.ParameterizedContext,
+  data: {
+    clients: { [key: string]: DVCClient };
+    users: { [key: string]: DVCUser };
+  }
+) => {
+  const body = ctx.request.body as LocationRequestBody;
+  const urlParts = ctx.request.url.split("/");
+  console.log(urlParts);
+  let entity = getEntityFromLocation(ctx.request.url, data);
+  console.log("entity", entity);
+
+  if (entity === undefined) {
+    ctx.status = 400;
+    ctx.body = {
+      errorCode: 400,
+      errorMessage: "Invalid request: missing entity",
+    };
+  } else if (body.command === undefined) {
+    ctx.status = 400;
+    ctx.body = {
+      errorCode: 400,
+      errorMessage: "Invalid request: missing command",
+    };
+  } else {
+    try {
+      const command = body.command;
+      const params: any | string | boolean | number = parseParams(
+        JSON.parse(body.params),
+        data
+      );
+
+      // TODO: handle async commands
+      // TODO: handle callback before invoking command
+      const resultData = entity[command](...params);
+
+      ctx.status = 200;
+      ctx.body = {
+        entityType: typeof resultData, // assuming this is the type of the entity for returned data
+        data: resultData,
+        logs: [], // TODO add logs here
+      };
+    } catch (error) {
+      if (body.isAsync) {
+        ctx.status = 500;
```

Comment:
The status here should still be 200 cause from the a testing perspective an error is a success cause the SDK did what it was supposed to do

Reply:
that is why i had it as 200, i changed it this morning :trollface: